OmniBLE/PumpManager/OmniBLEPumpManager.swift

```diff
@@ -450,8 +450,9 @@ extension OmniBLEPumpManager {
     }
 
     private var shouldWarnPodEOL: Bool {
+        let eolDisplayActiveTime = Pod.timeRemainingWarningThreshold + max(defaultExpirationReminderOffset, state.scheduledExpirationReminderOffset ?? 0.0)
```

Comment:
What's the use case behind taking the larger of these two?

Reply:
The use case for taking the max of the default and current notification offset:
* The default warning is applied to every pod
* For each pod, user can choose a different reminder that is applied just to that pod
* Example: for this pod, user wants to be notified 24 hours in advance, when normally they select no reminder
  * In that case, the HUD would start showing the lifecycle bar on the second day instead of when the warning happened.

Side note - that's why comments on PR should be via the PR - not PM.  I can't even remember who sent me the PM.  But if it had been posted here, we would all have seen it.

Reply:
Seems like the warning should be based off of scheduledExpirationReminderOffset, instead of the max of the two.  If they have one pod where they want the warning later than default, the default would override that.

Reply:
Via PM:
Pete suggests:
I think the change is to use state.scheduledExpirationReminderOffset, rather than max(defaultExpirationReminderOffset, state.scheduledExpirationReminderOffset ?? 0.0).
Marion will implement change and test.